SecurityPkg/Library/Tcg2PreUefiEventLogLibNull/Tcg2PreUefiEventLogLibNull.c

```diff
@@ -0,0 +1,20 @@
+/** @file Tcg2PreUefiEventLogLibNull.c
+NOTE: NULL Tcg2PreUefiEventLogLibNull library instance

```

Comment:
You can drop "NOTE:" here (and in the INF file) since a file description is expected anyway.

```suggestion
  NULL Tcg2PreUefiEventLogLibNull library instance
```

Reply:
Taken care